src/esp/gfx/replay/Recorder.cpp

```diff
@@ -208,6 +228,7 @@ RenderAssetInstanceState Recorder::getInstanceState(
 }
 
 void Recorder::updateInstanceStates() {
```

Comment:
Can you update this function name? Since you are updating instance states and also ... "rig instances" (?)